portfolio/src/main/webapp/script.js

```diff
@@ -65,17 +65,93 @@ for (var i = 0; i < btns.length; i++) {
     this.className += " active";
   });
 }
+function validateForm() {
+  var isEmptyName = document.forms["commentForm"]["name"].value;
+  var isEmptyEmail = document.forms["commentForm"]["email"].value;
+  var isEmptyComment = document.forms["commentForm"]["comment"].value;
 
-function getServerComments(){
-    fetch('/data')  // sends a request to /my-data-url
-.then(response => response.json()) // parses the response as JSON
-.then((comments) => { // now we can reference the fields in myObject!
-  console.log(comments.Femi);
-  console.log(comments.Dera);
-  console.log(comments.Kamsi);
-
-    const showComments = document.getElementById('comments-container');
-    showComments.innerHTML = 'Femi: ' + comments.Femi + ', Dera: ' + comments.Dera + ', Kamsi: ' + comments.Kamsi;
-});
+  if (isEmptyName == "" || isEmptyEmail== "" || isEmptyComment == "") {
+    alert("All fields must be completed!");
+    return false;
+  }
+}
+
+function loadComments() {
```

Comment:
have two functions that do the same thing is a code smell often. also, having a function named xxxx2 is also a code smell :) good naming is necessary for others to understand your code - which is one of the most important functions of any code.

Reply:
done